doc/release-notes.md

```diff
@@ -35,366 +34,30 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-Masternode Reward Location Reallocation (MN_RR) Hard Fork
----------------------------------------------------------
-The MN_RR hard fork, first included in Dash Core v20, will be activated after v21 is adopted by masternodes. This
-hard fork enables the major feature included in this release: Masternode Reward Location Reallocation. The activation
-will also initiate the launch of the Dash Platform Genesis Chain.
+Transaction Retrieval Bug
+-------------------------
 
-As the MN_RR hard fork is an [Enhanced Hard Fork](https://github.com/dashpay/dips/blob/master/dip-0023.md),
-activation is dependent on both masternodes and miners. However, as this hard fork was first implemented in v20, only
-masternodes are required to upgrade for the hard fork to activate. Once 85% of masternodes have upgraded to v21,
-an EHF message will be signed by the `LLMQ_400_85` quorum, and mined into a block. This signature serves as proof that 85% of
-active masternodes have upgraded.
+A bug that prevented retrieval of information for unmined transactions with clients that do not participate in address
+relaying due to partial misclassification as a block-only connection, has been resolved. This was identified by Dash
+mobile wallets not seeing transactions until they were mined. Kudos to @HashEngineering and @knst for finding and investigating this issue.
```

Comment:
```suggestion
SPV Client Bugfix
---------------------------------------------------------
Networking backports resulted in a regression where SPV / light nodes are incorrectly treated as block_only connections.
This resulted Dash Core v21 is not sending anything but blocks to light clients.
Kuddos to @HashEngineering and @knst for finding and
investigating this issue.
```

Reply:
The clients weren't fully treated as block-only (only one check was affected) nor did the bug exclusively affect light clients (it affected any client that didn't indicate it wanted to relay addresses).